docs/docs/cloud/how-tos/enqueue_concurrent.md

```diff
@@ -1,11 +1,29 @@
 ## Enqueue
 
-This notebook assumes knowledge of what double-texting is, which you can learn about in the [double-texting conceptual guide](https://langchain-ai.github.io/langgraph/cloud/concepts/#double-texting).
+This guide assumes knowledge of what double-texting is, which you can learn about in the [double-texting conceptual guide](https://langchain-ai.github.io/langgraph/cloud/concepts/#double-texting).
 
 The guide covers the `enqueue` option for double texting, which adds the interruptions to a queue and executes them in the order they are received by the client. Below is a quick example of using the `enqueue` option.
 
-First, let's import our required packages and instantiate our client, assistant, and thread.
 
+First, we will define a quick helper function for printing out JS model outputs (you can skip this if using Python):
+
+```js
+import { coerceMessageLikeToMessage } from "@langchain/core/messages"
+
+function prettyPrint(m) {
+  m = coerceMessageLikeToMessage(m);
+  let padded = " " + m._getType() + " ";
```

Comment:
i still think we don't need `coerceMessageLikeToMessage` only to use it for `m._getType`, can't we just do `m["type"]` for this? or are those two different?